packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -6,7 +6,7 @@ import ThemeContext from 'terra-theme-context';
 import memoize from 'memoize-one';
 import ResizeObserver from 'resize-observer-polyfill';
 import ContentContainer from 'terra-content-container';
-import { injectIntl, intlShape, FormattedMessage } from 'react-intl';
+import { injectIntl, FormattedMessage } from 'react-intl';
```

Comment:
Is `FormattedMessage` safe to use here? It was [removed explicitly](https://github.com/cerner/terra-core/pull/3285/files) in terra-core. Curious when it's okay to use

> Removed usage of `FormattedMessage` in favor of `injectIntl`